hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java

```diff
@@ -382,7 +386,7 @@ public void run() {
 
   private final LockManager lockManager = new LockManager(this);
 
-  private LoadBalancer balancer;
+  private RSGroupBasedLoadBalancer balancer;
```

Comment:
This balancer acts like old balancer when RSGroups is off? If so, just rename RSGroupBasedLB as LB?